bindings/experimental/distrdf/python/DistRDF/Backends/Spark/__init__.py

```diff
@@ -0,0 +1,10 @@
+def RDataFrame(*args, **kwargs):
```

Comment:
Here there are several possibilities:
1. We leave the `RDataFrame` factory method here, and we make this module (`Spark`) inject itself in `ROOT.RDF.Distributed`.
2. We create a new module `ROOT.RDF.Distributed.Spark` that we injected in `ROOT.RDF.Distributed`, plus we inject the `RDataFrame` function int this new module.

The advantage of 1 is simplicity, but the module would inherit the name it has in `DistRDF`, thus making that detail visible.